src/modules/movieCounter.js

```diff
@@ -0,0 +1,3 @@
+const movieCount = (movie) => movie.length;
```

Comment:
Good job with comment counter. 👍  But let's follow the requirements and use same counting method for movies also.
>Even if the API gives you these numbers, you will create a specific function to calculate these numbers in each page. 
<img width="1040" alt="Screenshot 2023-02-08 at 9 11 36 PM" src="https://user-images.githubusercontent.com/25887253/217621962-4b4830a1-1586-43d6-be89-e9f7592258d6.png">